docs/lake/design.md

```diff
@@ -647,7 +678,7 @@ to scale to arbitrarily large footprints as described earlier.
 
 ## Lake Introspection
 
-Commit history, meta data about segments, segment key spaces,
+Commit history, metadata about segments, segment key spaces,
```

Comment:
We used the single word "metadata" everywhere else in the doc, and that seems to be the standard convention. :man\_shrugging: